FreeRTOS/FreeRTOS-Kernel
# Add a list of third party tools

Description
-----------
Add a list of third party tools

Test Steps
-----------
NA.

Checklist:
----------


- [NA] I have tested my changes. No regression in existing tests.
- [NA] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1071

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
